source/tutorials.rst

```diff
@@ -0,0 +1,113 @@
+Tutorials
+=========
+
+
+Getting started with ``pytest_container``
+-----------------------------------------
+
+.. note::
+
+   The following tutorial demonstrates how to get started using
+   ``pytest_container`` for a python project and to test the project's
+   Command-line interface.
+
+
+The example uses  `poetry <https://python-poetry.org/>`_ to manage the Python
+dependencies. If the :file:`pyproject.toml` doesn't already exist, create it using the :command:`poetry init`  command.
+
+Start by adding ``pytest_container`` as a development dependency:
+
+.. code-block:: shell-session
+
+   ❯ poetry add --dev pytest_container
+
+
+Add `pytest-xdist
+<https://github.com/pytest-dev/pytest-xdist>`_, so that tests can be executed in
+parallel:
+
+.. code-block:: shell-session
+
+   ❯ poetry add --dev pytest-xdist
+
+
+To keep things tidy, create a directory called :file:`tests`  for your tests. Add an empty file called :file:`tests/__init__.py`. Create the :file:`tests/conftest.py` file with the following contents:
```

Comment:
"To keep things tidy" is a bit too abstract IMHO because it states an intention but not the effect. I would rephrase it as follows to put emphasis, that it's good to keep the tests atop of this framework in one single place, and that this place can be arbitrary. e.g.

```suggestion
It's highly recommended to create a new directory e.g. :file:`tests`  for your tests to keep things tidy. Add an empty file called :file:`tests/__init__.py`. Create the :file:`tests/conftest.py` file with the following contents:
```